json_serializable/lib/src/utils.dart

```diff
@@ -243,7 +255,10 @@ CtorData writeConstructorInvocation(
       writeableFields.toSet().difference(usedCtorParamsAndFields);
 
   var buffer = new StringBuffer();
-  buffer.write('new $className(');
+  //
+  // Generate the static factory method
```

Comment:
should this be a separate method named `generateStaticFactory`?

Reply:
This is an old comment. This does not create a factory. It creates an invocation. Will remove.